src/main/resources/com/google/api/codegen/java/grpc_package.snip

```diff
@@ -0,0 +1,138 @@
+@snippet generate(metadata)
```

Comment:
Maybe we should move all of the metadata files under something like com/google/api/codegen/java/grpc/ , because when we generate a metadata package for GAPIC clients, we might have files by the same name with different content (e.g. the PUBLISHING.md file).